src/NuGet/Microsoft.CodeAnalysis.CSharp.CodeStyle.nuspec

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="3.3">
+    <id>Microsoft.CodeAnalysis.CSharp.CodeStyle</id>
+    <description>
+      .NET Compiler Platform ("Roslyn") code style analyzers for C#.
+    </description>
+    <dependencies>
+    </dependencies>
+
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <version>$version$</version>
+    <authors>$authors$</authors>
+    <licenseUrl>$licenseUrl$</licenseUrl>
+    <projectUrl>$projectUrl$</projectUrl>
+    <releaseNotes>$releaseNotes$</releaseNotes>
+    <tags>$tags$</tags>
+    <serviceable>true</serviceable>
+    <developmentDependency>true</developmentDependency>
+  </metadata>
+  <files>
+    <file src="Dlls\CSharpCodeStyle\Microsoft.CodeAnalysis.CSharp.CodeStyle.dll" target="analyzers\dotnet\cs" />
+    <file src="Dlls\CSharpCodeStyle\Microsoft.CodeAnalysis.CSharp.CodeStyle.xml" target="analyzers\dotnet\cs" />
```

Comment:
I was thinking the same thing. I'm pretty sure the answer is we don't need them, so I'll go ahead and remove them now.